Azure/sonic-platform-common
# Add y-cable driver for simulated mux

#### Description


This PR added y-cable driver for simulated mux. This y-cable driver conforms with the requirement of https://github.com/Azure/SONiC/pull/757/files

#### Motivation and Context

After the abstract y-cable base class is added for multi-vendor y-cable support, we also need a y-cable driver for simulated mux used in lab testing.

#### How Has This Been Tested?

1. Update the xcvrd package in pmon docker to include the changes in https://github.com/Azure/sonic-platform-daemons/pull/197
2. Update the sonic_y_cable package in pmon docker to include the changes in this PR.
3. Create `/etc/sonic/config_db.json` on DUT.
4. Restart the pmon docker.
5. Run some dualtor test scripts.
6. Check mux status on dualtor testbed.

#### Additional Information (Optional)
